.github/workflows/release-series-images.yml

```
@@ -95,33 +80,20 @@ jobs:
         # which is required for build-push-action
         IFS=,
         echo "::set-output name=taglist::${tag_list[*]}"
-
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-
-    - name: Log in to Docker Hub
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
       with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Log in to OSG Harbor
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
-      with:
-        registry: hub.opensciencegrid.org
-        username: ${{ secrets.OSG_HARBOR_ROBOT_USER }}
-        password: ${{ secrets.OSG_HARBOR_ROBOT_PASSWORD }}
-
+        driver: docker
+    
     - name: Build and push Docker image
       uses: docker/build-push-action@v2.2.2
       with:
+        load: true
         push: ${{ github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/') }}
-        context: .
+        context: ./hosted-ce
+        file: ./hosted-ce/Dockerfile
```

```suggestion
        context: ./hosted-ce
        file: ./hosted-ce/Dockerfile
```

What images will be built for the `osg-ce-condor` jobs?